external/rapidjson/CMakeLists.txt.rapidjson.in

```diff
@@ -23,7 +23,7 @@ include(ExternalProject)
 ExternalProject_Add(rapidjson
   GIT_REPOSITORY    @OLP_SDK_CPP_RAPIDJSON_URL@
   GIT_TAG           @OLP_SDK_CPP_RAPIDJSON_TAG@
-  GIT_SHALLOW       1
+  GIT_SHALLOW       0
```

Comment:
is it necessary or just for debugging?

Reply:
Necessary for using commit hash. With git shallow enabled, only tags and branches are allowed